### ISSUE TITLE
Bot doesn’t properly censor multi-line text as spoiler

### ISSUE THREAD

**Issue by RyanCheddar** [Artraxon/unexBot#5]
If OP says `“The X does X”` it’s censored properly, but if OP has multiple lines like:

```
“The X does X.

Honestly pretty obvious”
```

The text doesn’t get censored properly and becomes:

```
”>!The X does X

Honestly pretty obvious!<“
```

Which doesn’t get censored properly in Reddit.

[Example](https://www.reddit.com/r/Unexpected/comments/dstwmc/now_thats_what_i_call_a_landing/f6rlw27/?utm_source=share&utm_medium=ios_app&utm_name=iossmf)

**Comment by Artraxon** [comment]
Hmm, that's an interesting bug, but I'm not quite sure what would be the best solution, since the problem lies in the spoiler tag itself.

Possible Solutions would be to either require people to not use newlines at all or to create multiple spoiler tags, although I think that the latter would be quite annoying.
I'll look into it a bit later since it's not a super critical bug (or maybe someone else has interest in implementing a quick fix)

**Comment by MingweiSamuel** [comment]
https://github.com/Artraxon/unexBot/blob/1015c7db11f10d85f768f3850aa8064779c0adba/src/main/kotlin/de/rtrx/a/SubmissionMonitor.kt#L153-L154

Basically need to replace multiple newlines with a single newline + two spaces, since spoiler tags work for newline but not new paragraph

**Comment by Tuggster** [comment]
Oh... my... god... absolutely everywhere I go, I encounter you. Who is this man?! (Your good ole Pal, Rethling, Holder Of The Medal of Crappy Off Bands)

**Comment by RyanCheddar** [comment]
:)

**Comment by drislands** [comment]
@Artraxon What about dividing every paragraph into a separately spoilered text? I don't know how it would be accomplished in Kotlin but surely there must be a method to divide strings by certain separators, in this case double-newline?

**Comment by MKMilton** [comment]
@MingweiSamuel + @drislands 
(TL;DR, fix tag nesting w/ 2 versions of search/replace)
Both seem to be correct? :smiley: I don't speak this language yet, and I don't know what format the OP submissions will be, but there are probably only a limited number of possible ways to send multi-line explanations, right? Can I help you test versions somehow? 

Is redacting a function of Reddit, or the bot?

**Comment by drislands** [comment]
@Artraxon I made a PR for a change I made that seems to fix it: #11 

A [reddit user](https://old.reddit.com/r/Unexpected/comments/jr184c/trying_to_catch_a_fish/gbrnga2/) pointed out that apparently doing the two-space newline trick lets you use a single spoiler tag instead of needing to add one to every single paragraph, so I added a `replace()` method call to the reddit comment-getter to fix it.

**Comment by Artraxon** [comment]
oof, sorry guys for not replying, i thought I'd get email notifications from this issue :/
I'll merge #11 , that should hopefully fix it
